defaults/defaults.go

```diff
@@ -0,0 +1,39 @@
+package defaults
+
+import "github.com/carlmjohnson/truthy/is"
+
+// GetFirst returns the first value in vs which is non-zero.
+func GetFirst[T comparable](vs ...T) (t T) {
```

Comment:
Go 1.22 will have cmp.Or (which I added 😊) that does this same thing, so it's kind of redundant to have it here. OTOH, FirstAny is still useful.